AirCasting/SessionViews/SessionCardView.swift

```diff
@@ -117,11 +117,15 @@ struct SessionCardView: View {
     }
 
     private func selectDefaultStreamIfNeeded(streams: [MeasurementStreamEntity]) {
-        if selectedStream == nil {
-            if let newStream = session.streamWith(sensorName: session.userInterface?.sensorName ?? "") {
-                return selectedStream = newStream
+        // AirBeam is giving us two streams at first (F, RH) when doing mobile active, which prevents us from switching to PM2.5 as selected stream.
+        // Solution below is not ideal, as in the future (this was written in 29/08/2022) the number of streams can change. Yet, it is working well - selecting PM2.5 as deafult stream for every session.
+        if streams.count == 5 || streams.count == 1 {
```

Comment:
1. Use `guard` whenever the `if` statement is the last statement in a scope. This way you'll avoid indentation. Here especially since you right away have a next `if` inside (which should be changed to `guard` too)
2. Why don't you do a lookup in this `streams` array instead? Pseudocode:
```Swift
if streams.contains(session.userInterface?.sensorName) { 
     selectedStream = session.userInterface?.sensorName 
} else if streams.contains(session.defaultStreamSelection()) {
    selectedStream = session.defaultStreamSensorName
} else {
    selectedStream = streams.first
}
```

Reply:
1. I will do that, thx!
2. Having this kind of solution, I think we will execute something every time this is called. The case here is to wait until we have all 5 streams so that we can make the right selection. If I am wrong please correct me

Reply:
@jankrzempek let's discuss after you're back from your PTO